website/src/pages/sandbox.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: ComposeDB API Sandbox
+description: Test out example GraphQL queries on ComposeDB to learn how to access documents by relations, how to filter, and more.
+hide_table_of_contents: true
+---
+
+```mdx-code-block
+import App from '../components/HomepageFeatures/App';
```

Comment:
Could this module be named more explicitly please? `App` usually represents the root of the React app, not a single component